content/docs/dvclive/api-reference/index.md

```diff
@@ -86,7 +92,14 @@ See `Live.log_plot()`.
 live.next_step()
 ```
 
-See `Live.next_step()` and `Live.set_step()` for details.
+See `Live.next_step()`.
+
+Under the hood, `Live.next_step()` calls `Live.make_summary()` and
+`Live.make_report()`.
+
+If you want to decouple the `step` update from the rest of the calls, you can
+manually modify the `Live.step` property and call `Live.make_summary()` /
+`Live.make_report()`.
```

Comment:
Non-blocking suggested rewording:

```suggestion
If you want to update the `step` to a different number or don't want to call
`Live.make_summary()` / `Live.make_report()`, you can manually modify the
`Live.step` property (for example, `live.step = 1`).
```

Also, why no mention of `make_checkpoint()`?

Reply:
> Why no mention of make_checkpoint()?

I'm also curious as to why this is not considered part of the https://dvc.org/doc/dvclive/api-reference#basic-workflow .